### DISCUSSION TITLE
Bootstrap CDN

### DISCUSSION THREAD

**Discussion by SiriusBlack0** [twbs/bootstrap#36911] [General] [answered]
If for some reason the servers are down and i am using the cdn link to style my html code what will happen ?

**Comment by wnunezc** [comment, marked as answer]
your website will stop working properly; If your html is very highly customized with bootstrap features it will even be unreadable. I never recommend using public CDNs, you never know when something will stop working or if by chance it is blocked by a firewall or security protection everything could be broken

**Reply by SiriusBlack0** [reply]
Thanks

**Comment by Nayab007** [comment]
I agree , I had the same problem with the CDN
